bedrock/firefox/views.py

```diff
@@ -528,5 +528,8 @@ class HelloStartView(LatestFxView):
     def get_context_data(self, **kwargs):
         ctx = super(HelloStartView, self).get_context_data(**kwargs)
         incoming = self.request.GET.get('incomingConversation') or 'none'
+        source = self.request.GET.get('utm_content') or 'none'
+        source = source.replace('hello-tour_OpenPanel_', '')
```

Comment:
Looks like the `incoming_conversation` variable does the same. I'd prefer both of these be reworked to be purely client side since they don't provide any value from the server.